flashcards/web/src/App.tsx

```diff
@@ -7,6 +7,7 @@ import FormikTest from "modules/genericTests/FormikTest/FormikTest";
 import TailwindTest from "modules/genericTests/TailwindTest/TailwindTest";
 import FakerTest from "modules/genericTests/FakerTest/FakerTest";
 import MaterialUITest from "modules/genericTests/MaterialUITest/MaterialUITest";
+import AddDecksForm from "./components/AddDecksForm";
```

Comment:
absolute imports

Reply:
Use absolute imports `import AddDecksForm from "modules/AddDeck/components"
Fix file structure, we're following [THIS](https://alexkondov.com/tao-of-react/#group-by-route-module), so group files by route, to make it easy to navigate in repo.

Test components should go they've been there to validate if everything was setup correctly, not to stay.